react-app/src/components/projects/ProjectDialog.tsx

```diff
@@ -148,11 +150,20 @@ export const ProjectGeneralInfoDialog = (props: IProjectGeneralInfoDialog) => {
     );
   }, [statusTypes, initialValues]);
 
+  useEffect(() => {
+    setApprovedStatus(lookupData?.ProjectStatuses?.find((a) => a.Name === 'Approved for ERP')?.Id);
```

Comment:
Is the name the best way to find this? I think we do this in a few places, but I wonder if that's opening ourselves to issues if the names of the statuses change. I almost feel like an enum would be safer.
And if it's always going to be a single value, why not just set it explicitly?

Reply:
Not sure what you mean by set it explicitly, but yeah I have thought before that finding by name may not really be ideal. Putting the names in an enum would be an improvement at least since we could at least change the names anywhere they're used with just one line if we ever need to. Ultimately we kinda have to hardcode something though for these sorts of cases, whether it's a numeric id or a name.